todos/src/components/Todos.js

```diff
@@ -48,13 +56,23 @@ function Todos() {
               completedDate={elem.completedDate}
               onEdit={elem.onEdit}
             />
+          );
         })}
       </div>
+      {loadingState && (
+        <img className="spinner" src={spinner} alt="Loging"></img>
+      )}
       {showEmptyListIcon && <EmptyTaskList />}
-      {lessThanListLength ? (
+      {searchValue.length === 0 ? (
+        lessThanFilteredTodos ? (
+          <LoadMoreBtn type={LOAD_MORE} />
+        ) : (
+          filteredTodos.length > PER_PAGE && <LoadMoreBtn type={SHOW_LESS} />
+        )
+      ) : lessThanSearchedTodos ? (
```

Comment:
This looks too complex at first sight. Can we simplify this?

Reply:
Sure @hhu-wtag Bhai we can. Doing it right now.